overlord/configstate/configcore/refresh.go

```diff
@@ -53,6 +54,15 @@ func reportOrIgnoreInvalidManageRefreshes(tr RunTransaction, optName string) err
 }
 
 func validateRefreshSchedule(tr RunTransaction) error {
+	maxInhibitionDaysStr, err := coreCfg(tr, "refresh.max-inhibition-days")
+	if err != nil {
+		return err
+	}
+	if maxInhibitionDaysStr != "" {
+		if n, err := strconv.ParseUint(maxInhibitionDaysStr, 10, 8); err != nil || (n < 1 || n > 14) {
```

Comment:
if we are making this settable we should maybe let people increase a bit as well,  maybe replace 14 with 21

Reply:
Done. Also set two constants for the minimum and maximum values.